documentation/typedoc_post_process.ts

```diff
@@ -0,0 +1,56 @@
+import * as fs from 'fs';
+import glob from 'glob';
+import path from 'path';
+import {print} from '../testing/helpers';
+
+const BaseDir = path.join(__dirname, '..');
+const TypeDocsRoot = path.join(BaseDir, 'docs/reference/sdk');
+const MarkdownPattern = path.join(TypeDocsRoot, '**/*.md');
+// Extracts the name from "# Name", "# Type: Name", and
+// "# Type: Name<Template Stuff>".
+const TitleRegex = /^# (?:.+?\:\s*)?(.*?)(?:<.+>)?$/m;
+
+async function main(): Promise<void> {
+  const files = await getFiles();
+  const promises = [];
+  for (const file of files) {
+    promises.push(process(file));
+  }
+  await Promise.all(promises);
+}
+
+async function getFiles(): Promise<string[]> {
+  return new Promise((resolve, reject) => {
+    glob(MarkdownPattern, {}, (err, files) => {
```

Comment:
I think you can maybe use `glob.sync()` to simplify this?

Reply:
Done. Wasn't sure if the sync methods were kosher.